examples/scale-deployment.js

```diff
@@ -0,0 +1,29 @@
+const k8s = require('@kubernetes/client-node');
+
+const kc = new k8s.KubeConfig();
+kc.loadFromFile('./config');
+
+const k8sApi = kc.makeApiClient(k8s.AppsV1Api);
+
+const targetDeploymentName = 'docker-test-deployment';
+
+async function scale(namespace, name, replicas) {
+  // find the particular deployment
+  const res = await k8sApi.listNamespacedDeployment(namespace);
```

Comment:
This should probably be `k8sApi.readNamespacedDeployent(namespace, name);` Then you don't need the `for` loop.